globals.go

```diff
@@ -143,6 +144,13 @@ var (
 	phre           = regexp.MustCompile(`\?`)
 	defaultCharset = C.ub2(0)
 
+	typeNill      = reflect.TypeOf(nil)
```

Comment:
typeNil ?

Reply:
Yeah, type of nil, right?

Reply:
Yes. please

Reply:
Oh, I see. typeNill -> typeNil, sure thing

Reply:
Changed